src/hooks/useErc20.ts

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react'
```

Comment:
I like more the structure of Uniswap, where for this kind of hooks they have them inside of state.

In this case would be `state/erc20/hooks.ts`

Reply:
Check https://github.com/gnosis/gp-ui/pull/130

If I get your point, in that PR all erc20 related hooks should be on /erc20/hooks then?
I'll wait for your input there.